<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/AdditiveManufacturing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenQMaZdaFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open QMaZda Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOptionsFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/AdditiveManufacturing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenOptionsFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Options Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMaZdaFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/StandaloneSoft/qmazda1902_win64a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpenImageFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/AdditiveManufacturing/set1sample2raw/ROI12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditROIFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/AdditiveManufacturing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Mz Feat. Out Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenRoiFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open ROI Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBatFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/AdditiveManufacturing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenBatFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenBatFolder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageFilePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+.tif</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateBatFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Create *.bat file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutBat">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>411</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFilePrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Out File Prefix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBatFileName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AM_R121S2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageFilePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image File pattern</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBatFileName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bat File Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditImageFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearOut">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReload">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutFilePrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AM_R121S2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditROIFileName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelROIFileName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI filename</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOptionsFileName">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>260</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MaZdaOptAllFeatD8.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelROIFileName_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Options filename</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRoiNameSameAsImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Roi Name Same As Image</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
